.github/workflows/deploy.yml

```diff
@@ -96,18 +97,31 @@ jobs:
         with:
           state: 'success'
           token: '${{ secrets.GITHUB_TOKEN }}'
-      - name: 'Fire Wildfly Operator examples build'
+      - name: Login Operator Quickstart quay.io registry
         if: success()
+        run: docker login -u="${QUICKSTARTS_ROBOT_NAME}" -p="${QUICKSTARTS_ROBOT_PASSWORD}" quay.io
```

Comment:
Does ${QUICKSTARTS_ROBOT_PASSWORD} end up the workflow log? It could just be safer to use the secrets.QUICKSTART_ROBOT_PASSWORD version so the GHA log filtering gets applied correctly just in case.

Reply:
> Does ${QUICKSTARTS_ROBOT_PASSWORD} end up the workflow log? 

No, they don't end up in the log. These are standard environment variables that are already initialized from the secrets at the beginning under env section: https://github.com/wildfly/wildfly-s2i/pull/156/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R9

It is not necessary to use the secret directly on the run section because the variable is already initialized in the env section. Personally, I see it as a less flexible way to work with the environment variables, but other parts of the code workflow were done as you mentioned, so it makes sense to follow up the same approach and avoid confusion.